node_modules/gh-auth/lib/oauth/dao.js

```diff
@@ -0,0 +1,422 @@
+/**
+ * Copyright (c) 2015 "Fronteer LTD"
+ * Grasshopper Event Engine
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+var _ = require('lodash');
+
+var DB = require('gh-core/lib/db');
+var DBUtil = require('gh-core/lib/db/util');
+var log = require('gh-core/lib/logger').logger('gh-auth-oauth');
+
+/**
+ * Initialize the indexes and constraints of the OAuth data model
+ *
+ * @param  {Function}   callback        Standard callback function
+ * @param  {Object}     callback.err    An error that occurred, if any
+ */
+var init = module.exports.init = function(callback) {
+    var sequelize = DB.getSequelize();
+
+    // We join and look up OauthClients on AppId
+    sequelize.query('CREATE INDEX oauth_clients_appid_idx ON "OauthClients" ("AppId");').complete(function(err) {
```

Comment:
I guess so, I just followed suit with what the user module was doing. But the user module is using a trigram thingy which explains that.